Mopups/Mopups.Maui/Platforms/Android/Impl/AndroidMopups.cs

```diff
@@ -70,7 +89,44 @@ public Task RemoveAsync(PopupPage page)
         return Task.CompletedTask;
     }
 
-    Task<bool> PostAsync(Android.Views.View nativeView)
+    static void HandleAccessibility(ImportantForAccessibility importantForAccessibility)
+    {
+        Page? mainPage = Application.Current?.MainPage;
+
+        if (mainPage is null)
+        {
+            return;
+        }
+
+        int navCount = mainPage.Navigation.NavigationStack.Count;
+        int modalCount = mainPage.Navigation.ModalStack.Count;
+
+        Android.Views.View? platformMainPage = mainPage.Handler?.PlatformView as Android.Views.View;
+        if (platformMainPage is not null)
+        {
+            platformMainPage.ImportantForAccessibility = importantForAccessibility;
+        }
+
+        if (navCount > 0)
+        {
+            Android.Views.View? currentPage = mainPage.Navigation?.NavigationStack[navCount - 1]?.Handler?.PlatformView as Android.Views.View;
+            if (currentPage is not null)
+            {
+                currentPage.ImportantForAccessibility = importantForAccessibility;
```

Comment:
Could we switch this to using ?. instead of the if statement

Reply:
Same as previous comment